service-data-access/build.gradle

```diff
@@ -98,11 +98,6 @@ task integrationTest(type: Test) {
     }
 }
 
-docker {
-    files "build/libs/${bootJar.archiveFileName.get()}", "bin/entrypoint.sh"
-    buildArgs([JAR_FILE: bootJar.archiveFileName.get(), ENTRYPOINT_FILE: "entrypoint.sh"])
-}
-
```

Comment:
This common code has been moved to https://github.com/department-of-veterans-affairs/abd-vro/pull/531/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R104-R110